apps/api/src/app/content-templates/usecases/parse-preview/preview-email.command.ts

```diff
@@ -0,0 +1,11 @@
+import { IsDefined, IsString } from 'class-validator';
+import { IEmailBlock } from '@novu/dal';
+import { EnvironmentWithUserCommand } from '../../../shared/commands/project.command';
+
+export class PreviewEmailCommand extends EnvironmentWithUserCommand {
+  @IsDefined()
+  content: string | IEmailBlock[];
+
+  @IsString()
+  contentType: 'editor' | 'customHtml';
```

Comment:
I had made a type to hold this time ago. It lives here:
https://github.com/novuhq/novu/blob/271488a5ea716f00167d397c01694e8287d5c289/libs/shared/src/entities/message-template/message-template.interface.ts#L3

Reply:
I do think it can even be converted to enum values the values.